client/src/controllers/CountController.ts

```diff
@@ -0,0 +1,64 @@
+import { Controller } from '@hotwired/stimulus';
+import { ngettext } from '../utils/gettext';
+
+export class CountController extends Controller<HTMLFormElement> {
+  static targets = ['total', 'label'];
+  static values = {
+    containerBody: { type: String, default: 'body' },
+    min: { type: Number, default: 0 },
+    target: {
```

Comment:
Maybe we should call this `find` or something, using `target` is a bit confusing.

This way we can use the code roughly like this - which reads a bit better for WHERE we are looking vs WHAT we are looking for.

```
this.totalValue = document.querySelector(this.containerValue).querySelectorAll(this.findValue).length;
```

Note: The code above wont work as is, but just an idea.